webpack.config.production.js

```diff
@@ -1,14 +1,24 @@
 var webpack = require('webpack');
+var InlineManifestWebpackPlugin = require('inline-manifest-webpack-plugin');
 var config = require('./webpack.config.base');
 
 config.bail = true;
 config.profile = false;
 config.devtool = 'source-map';
+config.output.filename = '[name].[chunkhash].js';
```

Comment:
Just played with this with `hash: true` and it has the same problem of the hash is unique per build, but not per chunk that's built—reducing the effectiveness of the caching.